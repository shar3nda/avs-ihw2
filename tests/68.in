7XA=}v{3Z9J9!]qw=/<)6\c!n%f4?3zU[$6'<YazWV8S| z6A0(;UQqMo.T~lBzqj{vyI$