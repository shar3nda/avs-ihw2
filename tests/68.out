Lowercase: 823, uppercase: 820